module/window_swallowing.lua

```diff
@@ -12,11 +12,12 @@ local window_swallowing_activated = false
 
 -- you might want to add or remove applications here
 local parent_filter_list = beautiful.parent_filter_list
+    or beautiful.dont_swallow_classname_list
     or { "firefox", "Gimp", "Google-chrome" }
 local child_filter_list = beautiful.child_filter_list
-    or { }
+    or beautiful.dont_swallow_classname_list or { }
```

Comment:
Just a thought I have reading this : There is a corner case where the user can require the module before calling `beautiful.init`. In this situation, the user defined lists in their theme will be ignored because these variables here are global to the module and initialized only here, at the module first load.

Reply:
that is a corner case with all of bling and the docs specify requiring bling after beautiful.init

https://blingcorp.github.io/bling/#/?id=installation